### PR TITLE
VA Profile disability rating

### PR DESCRIPTION
**Pull request by rmtolmach**
## Summary

- This disability model is needed to grab `combined_service_connected_rating_percentage` from the `disabilityRating` bio in VA Profile. It's needed to create all of the military_information needed to prefill forms. 
- This work was split off of https://github.com/department-of-veterans-affairs/vets-api/pull/14184.

## Related issue(s)
- This work is part of: https://github.com/department-of-veterans-affairs/va.gov-team/issues/67283
- Which is a part of this: https://github.com/department-of-veterans-affairs/va.gov-team/issues/61657
- Which is part of the epic: https://github.com/department-of-veterans-affairs/va.gov-team/issues/44853
- Manifest repo changes: 
    - https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2345
    - https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2346

## What areas of the site does it impact?
* This is new functionality that won't be called until https://github.com/department-of-veterans-affairs/vets-api/pull/14184 is merged. 

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by tpharrison — DISMISSED** [review]

File: lib/va_profile/models/disability.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module VAProfile
+  module Models
+    class Disability < Base
+      attribute :combined_service_connected_rating_percentage, String
```
Comment (rmtolmach) [inline lib/va_profile/models/disability.rb:8]: @LindseySaari @rjohnson2011 I saw that Ryan used `validates` for the status data here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/va_profile/models/veteran_status.rb#L11
Do we also need that?

Comment (LindseySaari) [reply lib/va_profile/models/disability.rb:8]: + to this yes :)

Comment (rmtolmach) [reply lib/va_profile/models/disability.rb:8]: hmm... The veteran_status model is also slightly different in that it doesn't have error handling for a `nil` value, like we do in this model:

```ruby
      def self.build_disability_rating(rating)
        return nil unless rating

        VAProfile::Models::Disability.new(
          combined_service_connected_rating_percentage: rating
        )
      end
```
@RachalCassity do you think we still need `validates :combined_service_connected_rating_percentage, presence: true, length: { maximum: 25 }`?

Comment (RachalCassity) [reply lib/va_profile/models/disability.rb:8]: Yes

Comment (RachalCassity) [reply lib/va_profile/models/disability.rb:8]: All validations should be moved over to the new files

Comment (RachalCassity) [reply lib/va_profile/models/disability.rb:8]: I can't find` validates :combined_service_connected_rating_percentage, presence: true, length: { maximum: 25 }` anywhere in master

Comment (rmtolmach) [reply lib/va_profile/models/disability.rb:8]: Fixed! 🎉

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**